SubEthaEdit-Mac/Source/PlainTextDocument.m

```diff
@@ -3578,6 +3617,7 @@ - (BOOL)writeToURL:(NSURL *)absoluteURL ofType:(NSString *)inType forSaveOperati
 		}
 		//        NSArray *xattrKeys = [UKXattrMetadataStore allKeysAtPath:[absoluteURL path] traverseLink:YES];
 		//        NSLog(@"%s xattrKeys:%@",__FUNCTION__,xattrKeys);
+        I_fileHash = [writtenData md5Data];
```

Comment:
Do we want to update this for all the cases here? E.g. autosaving should not update it, save as should not either. Those things should be conveyed through the save operation type if I remember correctly.